docs/onboarding_to_cluster.md

```diff
@@ -0,0 +1,151 @@
+# Onboarding to a cluster
+
+Currently we support 2 ways to request/perform onboarding to any supported clusters:
+
+1. Onboarding request - please use the **Onboarding to Cluster** issue template in [operate-first/support](https://github.com/operate-first/support) repository.
+2. Opening a PR with the desired changes. The rest of this doc focuses on this second option. We'd gladly welcome your contributions.
+
+## Prerequisites
+
+1. Please install our [toolbox](https://github.com/operate-first/toolbox) to have the developer setup ready automatically for you.
+2. Or you can install the tools manually. You'll need [kustomize](https://kustomize.io/), [sops](https://github.com/mozilla/sops) and [ksops](https://github.com/viaduct-ai/kustomize-sops).
+
+Please fork/clone the [operate-first/apps](https://github.com/operate-first/apps) repository. **During this whole setup, we'll be working within this repository.**
+
+## Recipe
+
+If you want to know more about the overall design please consult the ADR documentation at [operate-first/support](https://github.com/operate-first/blueprint).
+
+In general we store all the cluster-scoped resources in a `cluster-scope` kustomize application within this repository.
+
+## Adding namespaces
+
```

Comment:
Can we add a comment about, when picking a namespace, use a prefix for all their namespaces that's unique to their team. This will make it easier for argocd multi tenancy later.